src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/hiddenfield/Example1.java

```diff
@@ -0,0 +1,29 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="HiddenField"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.hiddenfield;
+
+// xdoc section -- start
+abstract class Example1 {
```

Comment:
lets remove `abstract` as it looks like it is only target of this Check.

but add inner class
```
...
 Example1 setField(String field) { // violation, ''field' hides a field'
    this.field = field;
    return null;
  }
  class Inner {
    abstract int method(String field); // OK, because it match ignoreFormat
  }
}
```
please keep Inner in all Examples.
Examples should be same by java code and only comments content is changing.

Reply:
I think it is last point to improve.
But please after fix please double check that all test as same by size and structure and only comment content is changing. Please look at images in issue description, they show comparison.

Reply:
Should I make the Inner class abstract since the abstract method need to be inside a abstract class.

Reply:
<img width="761" alt="Screenshot 2024-11-01 at 7 16 04 PM" src="https://github.com/user-attachments/assets/2ce20a12-c5da-4341-bd15-baff860f69bc">
Is this structure fine for all Examples?

Reply:
Done.